policy/release/test.rego

```diff
@@ -91,32 +91,79 @@ deny contains result if {
 }
 
 # METADATA
-# title: All required tests passed
+# title: No tests failed
 # description: >-
-#   Confirm that all the tests in the test results
-#   have a successful result. A successful result is one that isn't a
-#   "FAILURE" or "ERROR". This will fail if any of the tests failed and
-#   the failure message will list the names of the failing tests.
+#   Produce a violation if any non-informative tests have their result set to "FAILED".
+#   The result type is configurable by the "failed_tests_results" key, and the list
+#   of informative tests is configurable by the "informative_tests" key in the rule data.
 # custom:
-#   short_name: required_tests_passed
-#   failure_msg: "Test %q did not complete successfully"
+#   short_name: no_failed_tests
+#   failure_msg: "Test %q failed"
```

Comment:
Observation: I was wondering if we'd need a transition period for anyone addressing the `required_tests_passed` rule, I guess not really but it is an interesting question that we might want to consider. What happens when we remove a rule? Do we have any compatibility guarantees that we need to uphold?

Reply:
Yeah. It's an interesting one. It would be interesting to see if we can provide aliases for policy rules. Filed https://github.com/enterprise-contract/ec-policies/issues/795 as a follow up.